package.json

```diff
@@ -64,26 +67,28 @@
     "babel-plugin-transform-object-rest-spread": "^6.26.0",
     "babel-polyfill": "6.26.0",
     "babel-preset-env": "1.6.0",
+    "babel-preset-flow": "^6.23.0",
```

Comment:
Please don't use version ranges for dev dependencies … at iohk we are updating dependencies conservatively by hand. Same thing for the new / updated deps below.